app/assets/javascripts/application.js

```diff
@@ -14,6 +14,7 @@
 //= require jquery_ujs
 //= require foundation
 //= require d3
+//= require skrollr
```

Comment:
Should change this so it only loads for `about` since we're not using it anywhere else.